pkg/csi_driver/multishare_ops_manager_test.go

```diff
@@ -30,13 +30,15 @@ import (
 )
 
 const (
-	testInstanceScPrefix = "testinstancescprefix"
+	testInstanceScPrefix = "test-prefix"
 	testInstanceName     = "testInstanceName"
 	testShareName        = "testShareName"
+	testVPCNetwork       = "testSharedNetwork"
 )
 
 var (
 	testInstanceHandle = fmt.Sprintf("%s/%s/%s", testProject, testRegion, testInstanceName)
+	testRegions        = []string{"us-central1"}
```

Comment:
why hard code region here but use `testRegion` everywhere else

Reply:
Fixed. It should be replaced with `testRegion`.